README.md

```diff
@@ -1,17 +1,21 @@
-# Efficient HMC implementations in Julia
+# AdvancedHMC.jl: a robust, modular and efficient implementation of advanced HMC algorithms
 
-[![Build Status](https://travis-ci.org/TuringLang/AdvancedHMC.jl.svg?branch=master)](https://travis-ci.org/TuringLang/AdvancedHMC.jl) [![Coverage Status](https://coveralls.io/repos/github/TuringLang/AdvancedHMC.jl/badge.svg?branch=kx%2Fbug-fix)](https://coveralls.io/github/TuringLang/AdvancedHMC.jl?branch=kx%2Fbug-fix)
+[![Build Status](https://travis-ci.org/TuringLang/AdvancedHMC.jl.svg?branch=master)](https://travis-ci.org/TuringLang/AdvancedHMC.jl)
+[![DOI](https://zenodo.org/badge/72657907.svg)](https://zenodo.org/badge/latestdoi/72657907)
+[![Coverage Status](https://coveralls.io/repos/github/TuringLang/AdvancedHMC.jl/badge.svg?branch=kx%2Fbug-fix)](https://coveralls.io/github/TuringLang/AdvancedHMC.jl?branch=kx%2Fbug-fix)
 
-**The code from this repository is used to implement HMC samplers in [Turing.jl](https://github.com/TuringLang/Turing.jl).**
+AdvancedHMC.jl is originally designed to implement HMC samplers in [Turing.jl](https://github.com/TuringLang/Turing.jl), a probabilistic programming language (PPL) in Julia. 
```

Comment:
```suggestion
`AdvancedHMC.jl` is part of [Turing.jl](https://github.com/TuringLang/Turing.jl), a probabilistic programming library in Julia. If you are interested in using `AdvancedHMC.jl` through a probabilistic programming language, please check it out!
```

I made a minor update to Hong's suggestion, since "please check it" seemed strange to me.